examples/common/text_data.py

```diff
@@ -167,7 +153,6 @@ def build_text_dataloader(cfg: DictConfig, device_batch_size: int):
         local=cfg.dataset.local,
         tokenizer_name=cfg.dataset.tokenizer_name,
         max_seq_len=cfg.dataset.max_seq_len,
-        group_method=cfg.dataset.group_method,
         remote=cfg.dataset.get('remote', None),
         split=cfg.dataset.get('split', None),
         shuffle=cfg.dataset.get('shuffle', False),
```

Comment:
`keep_zip` default should be `None` in keeping with the default [here](https://streaming.docs.mosaicml.com/en/stable/api_reference/generated/streaming.StreamingDataset.html#streamingdataset) and [here](https://github.com/samhavens/benchmarks/blob/R-387--update-scripts-to-pre-concat/examples/common/text_data.py#L57)

Reply:
@abhi-mosaic said "Now that we have streaming==0.2.2, the default for num_canonical_nodes should probably be made high (128 or 256 or so) to establish good shuffling randomness." Not sure if we should set it to 128 everywhere now...?

Reply:
The 128 is quite important for good shuffling, and I did some testing to pick a value that did not add too much overhead on init (<15s): https://github.com/mosaicml/examples/pull/123#issue-1573524320

It is probably a good idea for Streaming team to set a default large number too... I will make a JIRA for them.

The other defaults, I guess it's fine to copy them over? Is it also a good idea to put the 128 value in the default of our `StreamingText` class? Maybe

Reply:
set `keep_zip` to `None`, defaulted `num_canonical_nodes` to 128 everywhere, change `shuffle_seed` default to 9176.